shared/AppInsightsCommon/src/Util.ts

```diff
@@ -542,7 +517,7 @@ export class Util {
      * Returns string representation of an object suitable for diagnostics logging.
      */
     public static dump(object: any): string {
-        const objectTypeDump: string = Object.prototype.toString.call(object);
+        const objectTypeDump: string = Object[prototype].toString.call(object);
```

Comment:
Using index lookup (array) to support better minification of the resulting JS code.